localstack/utils/container_utils/docker_cmd_client.py

```diff
@@ -660,6 +660,11 @@ def start_container(
         LOG.debug("Start container with cmd: %s", cmd)
         return self._run_async_cmd(cmd, stdin, container_name_or_id)
 
+    def attach_to_container(self, container_name_or_id: str):
+        cmd = self._docker_cmd() + ["attach", container_name_or_id]
+        LOG.debug(f"Attaching to container {container_name_or_id}")
```

Comment:
nit: don't use f-strings with loggers
```suggestion
        LOG.debug("Attaching to container %s", container_name_or_id)
```

Reply:
1) it's also a convention with `logging` we follow throughout the codebase :-)
2) the reason is that with `LOG.debug("Attaching to container %s", container_name_or_id)`, string creation is deferred until the log message is actually printed. if you make a `debug` call, but the log level is `info` or higher, the string will never actually be formatted. whereas with f-strings you're always creating the string even though you may not need it.

Reply:
Ok great, those are some good reasons. Thanks!